performance.rst

```diff
@@ -122,6 +122,10 @@ The preload file path is the same as the compiled service container but with the
     ; php.ini
     opcache.preload=/path/to/project/var/cache/prod/srcApp_KernelProdContainer.preload.php
 
+Use the :ref:`container.no_preload <dic-tags-container-nopreload>` and
+:ref:`container.preload <dic-tags-container-preload>` service tags to define
+which classes should or should not be preloaded PHP.
```

Comment:
As you put first the no preload, maybe better to write should not or should (in same order) wdyt?